src/fastman/commands/misc.py

```
@@ -33,41 +33,117 @@ def handle(self):
+    print(json.dumps(routes_data))
+except Exception as e:
+    print(json.dumps({"error": str(e)}))
+    sys.exit(1)
+"""
+        script_path = Path("_fastman_routes.py")
+        PathManager.write_file(script_path, script_content, overwrite=True)
+
+        try:
+            manager_prefix = PackageManager.get_run_prefix()
+            if manager_prefix:
+                cmd = manager_prefix + ["python", str(script_path)]
```

Ruff (0.14.14)

[warning] Consider `[*manager_prefix, "python", str(script_path)]` instead of concatenation

Replace with `[*manager_prefix, "python", str(script_path)]`

(RUF005)